### PR TITLE
tts-adapter-sapinative : support for microsoft natural voices over NaturalVoicesSAPIAdapter

### PR DESCRIPTION
**Pull request by NPavie**
This PR aims to allow the use of Microsoft natural voices with the sapinative adapter, when exposed to the SAPI TTS engine with the NaturalVoicesSAPIAdapter :
- The mutex lock is extended to avoid a timeout due to longer synthesis time with those voice
- bookmark events monitoring is replaced by evaluating ssml marks position between word boundaries event and after the "end of input stream" event, as to circumvent a bug in marks reporting by natural voices

Additionally, the PR also
- increments the targeted Windows SDK to its version 10.0.22621.0
- Remove the space added when unwrapping SSML tokens, that would lead to mispronunciation in some cases

This PR should fix daisy/pipeline#784

The adapter has been tested in production with the Daisy Pipeline App 1.6.0 beta 1, on the hauy_valid.xml sample, with NaturalVoicesSAPIAdapter version 0.2.

### CODE REVIEW

**Comment by bertfrees** [comment]
Merged!

File: tts/tts-adapters/tts-adapter-sapinative/src/main/resources/transform-ssml.xsl [hunk 1]
```diff
@@ -54,9 +54,6 @@
   <xsl:template mode="copy" match="token">
     <!-- tokens are unwrapped because they are not SSML1.0-compliant and not SAPI-compliant-->
     <xsl:apply-templates select="node()" mode="copy"/>
-    <xsl:if test="following-sibling::*">
-      <xsl:text> </xsl:text>
-    </xsl:if>
```
Comment (bertfrees) [inline tts/tts-adapters/tts-adapter-sapinative/src/main/resources/transform-ssml.xsl:59]: @NPavie This change seems to have broken a test (`exampleSSML` in `SapiSSMLTest`). It slipped under my radar. Can you have a look?

**Comment by NPavie** [comment]
hum that's strange, in my setup it was passing the test.
I'll retest with the latest changes and clean maven repo just in case.

**Comment by NPavie** [comment]
Oh yes, I see, I had deactivated the test in my setup but forgot to update it, sorry.
Given the ssml tested in here, this should not have impact on the production side.

The ssml tested here has no spaces between tokens (meaning no spaces between word).
There are cases where words/tokens can be juxtaposed and should not be separated by a space, or it could break pronunciation.
Like `<token>d'</token><token>autres</token>` or `<token>d</token>'<token>autres</token>` (i don't remember right now if the tokenizer includes or not the apostrophe in the token) should not have spaces inserted or it will pronounced "D autres" (was the case in production before I removed the space inserted by the stylesheet).

From my tests in production, the pipeline does produce tokenized sentences with spaces correctly preserved between tokens, so I assumed this test was to verify that a problem with the ssml received by the adapter was bypassed at some point.

**Comment by bertfrees** [comment]
So what would be a correct test? Do there need to be spaces between the tokens?

**Comment by NPavie** [comment]
In this particular test, I think so.

**Comment by bertfrees** [comment]
OK, I'll change that.

**Pull request merged**